### PR TITLE
Make bain SEM work again with updated SEM component

### PR DESCRIPTION
**Pull request by koenderks**
Closes https://github.com/jasp-stats/jasp-test-release/issues/1249

+ I did a Ctrl+I on the qml file

### CODE REVIEW

File: R/bainsembayesian.R [hunk 1]
```diff
@@ -79,7 +74,7 @@ BainSemBayesian <- function(jaspResults, dataset, options, ...) {
   usedvars <- usedvars[order(nchar(usedvars), decreasing = TRUE)]
   withSingleQuotes <- paste("\\b'", usedvars, "'\\b", sep="")
   withDoubleQuotes <- paste('\\b"', usedvars, '"\\b', sep="")
-  newNames <- usedvars
+  newNames <- .v(usedvars)
```
Comment (vandenman) [inline R/bainsembayesian.R:77]: are you sure you need `.v` here?

Comment (koenderks) [reply R/bainsembayesian.R]: You are right, we don't need it.

**Comment by vandenman** [comment]
Okay so the analysis works again, so I think this can be merged. Two issues though:

1. The unit tests don't work
2. The upgrader doesn't work:
image

if you open 
image

@koenderks can you open an issue for this? Note that you can probably reuse most of what we'll do for jaspSEM because there we have the exact same issue.

**Review by vandenman — APPROVED** [review]

**Comment by koenderks** [comment]
This is the first version where bain sem is released though, so no one has any bain SEM files yet. Maybe just remaking the JASP file is already enough?

**Comment by vandenman** [comment]
yes, that would suffice.

**Pull request merged**